buildSrc/src/main/groovy/com/mobbeel/plugin/task/CopyDependenciesTask.groovy

```diff
@@ -186,8 +191,8 @@ class CopyDependenciesTask extends DefaultTask {
         println()
     }
 
-    def processZipFile(File aarFile, Dependency dependency) {
-        String tempDirPath = "${temporaryDir.path}/${dependency.name}_zip"
+    def processZipFile(File aarFile, String dependencyName, String dependencyGroup, String dependencyVersion) {
```

Comment:
I don't understand what it bring up to change the signature of this method.

Reply:
Its because we don't have access to "Dependency" object in its new call in processPomFile